std/process.d

```diff
@@ -3084,6 +3084,48 @@ static:
     }
 
     /**
+    Identify whether a variable is defined in the environment.
+
+    Because it doesn't return the value, this function is cheaper than `get`.
+    However, if you do need the value as well, you should just check the
+    return of `get` for `null` instead of using this function first.
+
+    Example:
+    -------------
+    // good usage
+    if ("MY_ENV_FLAG" in environment)
+        doSomething();
+
+    // bad usage
+    if ("MY_ENV_VAR" in environment)
+        doSomething(environment["MY_ENV_VAR"]);
+
+    // do this instead
+    if (auto var = environment.get("MY_ENV_VAR"))
+        doSomething(var);
+    -------------
+    */
+    bool opBinaryRight(string op : "in")(in char[] name) @trusted
+    {
+        version (Posix)
+            return core.sys.posix.stdlib.getenv(name.tempCString()) !is null;
+        else version (Windows)
+        {
+            SetLastError(NO_ERROR);
+            if (GetEnvironmentVariableW(name.tempCStringW, null, 0) > 0)
+                return true;
+            immutable err = GetLastError();
+            if (err == ERROR_ENVVAR_NOT_FOUND)
+                return false;
+            // some other windows error. Might actually be NO_ERROR, because
+            // GetEnvironmentVariable doesn't specify whether it sets on all
+            // failures
+            throw new WindowsException(err);
+        }
+        else static assert(0);
```

Comment:
Does phobos even support anything other than posix or windows? I only put these in as sanity checks for future porters.